updater/lib/dependabot/dependency_group_engine.rb

```diff
@@ -21,6 +21,22 @@ class DependencyGroupEngine
     class ConfigurationError < StandardError; end
 
     def self.from_job_config(job:)
+      if job.security_updates_only? && job.dependencies && job.dependencies.count > 1 && job.dependency_groups.empty?
```

Comment:
should this be refactored into a function: `is_grouped_security_update`?

Reply:
It's kind of janky because this is a class method. I'd have to define it on `Job` for it to make sense, but that seems really unclear to me: `Job.grouped_security_update?` will have a changing meaning as we move forward. I think we should leave this ugly and then remove it when the API starts sending groups.